### PR TITLE
Adding love_post.rb file part of hw4

### PR DESCRIPTION
**Pull request by djuare**

### CODE REVIEW

File: DavidJuarez-djuare/hw_5/mashable.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+end
+
+
+mashable_url = "http://mashable.com/stories.json" 
+mashable_json_response = connect_to_api(mashable_url)
+stories = find_stories(mashable_json_response)
+print_stories(stories)
+
+
```
Comment (Borgaard) [inline DavidJuarez-djuare/hw_5/mashable.rb]: Concise and effective code!

File: DavidJuarez-djuare/hw_6/rise_of_robots.rb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    #This method creates new instances of the robot class
+    # instance variables are available through out the class.
+    # @name is an instance_variable
+    # basic constructor below
+    def initialize(name, type, origin)
+      @name = name #instance variable, always start with @
+      @type = type #instance variable
+      @origin = origin #instance variable
+
```
Comment (Borgaard) [inline DavidJuarez-djuare/hw_6/rise_of_robots.rb]: This is _excellent_ commenting practice, David!

File: DavidJuarez-djuare/hw_6/rise_of_robots.rb [hunk 3]
```diff
@@ -0,0 +1,108 @@
+require 'pry'
+require 'pry-byebug'
+
+
+  class Robot #Classes are never plural
+    #constructor - a way to create an instance of a class
```
Comment (Borgaard) [inline DavidJuarez-djuare/hw_6/rise_of_robots.rb]: Yes! :+1:

File: DavidJuarez-djuare/hw_4/love_post.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    when "no"
+      puts "Ok. Call soon. You love #{love_interest}"
+    else
+      puts "That's not a valid answer \n"
+      puts "Please answer 'Yes' or 'No' \n"
+      capture_love_interest_response(love_interest)
+    end
+end
+
+### This is where we call methods to run our script
```
Comment (Borgaard) [inline DavidJuarez-djuare/hw_4/love_post.rb]: And it runs!

File: DavidJuarez-djuare/hw_4/car_lot.rb [hunk 5]
```diff
@@ -56,8 +56,9 @@ def add_cars(array, *all_cars)
 
 
 cars = []
-add_cars(cars, tesla, porsche, toyota, bmw, audi)
-#cars.push(tesla, porsche, toyota, bmw, audi)
+array = []
+add_cars(array, tesla, porsche, toyota, bmw, audi)
+cars.push(tesla, porsche, toyota, bmw, audi)
```
Comment (Borgaard) [inline DavidJuarez-djuare/hw_4/car_lot.rb]: :+1:

File: DavidJuarez-djuare/hw_5/mashable-dj-wip.rb [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  puts new_rising
+end
+# Rising
+
+# hot
+def create_rising_hash(story)
+  new_hot = {category: story["channel_label"], title: story["title"], upvotes: story["shares"]["total"]}
+  puts new_hot
+end
+# hot
```
Comment (Borgaard) [inline DavidJuarez-djuare/hw_5/mashable-dj-wip.rb]: Are these two the same functions?

Comment (djuare) [inline DavidJuarez-djuare/hw_5/mashable-dj-wip.rb]: Hi Nicole,

I am trying to obtain two more arrays, create their corresponding hashes hot and new_rising to get the same result as new array.

I will schedule time to meet to go over some questions. Please let me know if I am missing homework or if I am falling behind.

Thanks.

Kind regards,
David Juarez

File: DavidJuarez-djuare/hw_4/car_lot.rb (hunk 5 above)
Comment (djuare) [inline DavidJuarez-djuare/hw_4/car_lot.rb]: 👍

File: DavidJuarez-djuare/hw_4/love_post.rb (hunk 4 above)
Comment (djuare) [inline DavidJuarez-djuare/hw_4/love_post.rb]: Does it run?? 😄👍

File: DavidJuarez-djuare/hw_6/rise_of_robots.rb (hunk 2 above)
Comment (djuare) [inline DavidJuarez-djuare/hw_6/rise_of_robots.rb]: Can I push more programs to this folder?

File: DavidJuarez-djuare/hw_5/mashable.rb (hunk 1 above)
Comment (djuare) [inline DavidJuarez-djuare/hw_5/mashable.rb]: Thank very very very much for all your help, time and considerations.

Regards,
David Juarez

**Pull request merged**